spec/helpers/application_helper_spec.rb

```diff
@@ -151,4 +151,24 @@
       expect(result).to have_link(text: /articles/, href: '/articles?q=my+query&search_field=title')
     end
   end
+
+  # TODO: Add bento search link
```

Comment:
I see this sprinkled througout this PR, but I'm not sure what it means.  Can you add a link to an issue that explains more?

Reply:
✅ added an issue number (#1695) to the comment. This will be resolved by the end of the work cycle.